File: Excel/601 Challenge.py
import pandas as pd

path = "601 Count Sorted Words.xlsx"
input = pd.read_excel(path, usecols="A:C", skiprows=1, nrows=10)
test = pd.read_excel(path, usecols="E:F", skiprows=1, nrows=3).squeeze()

result = input.apply(lambda col: sum(list(x) == sorted(x) for x in col)).to_frame().reset_index()
result.columns = test.columns

print(test.equals(test))  # True

File: Excel/600-699/601/601 Challenge.R
library(tidyverse)
library(readxl)

path = "Excel/601 Count Sorted Words.xlsx"
input = read_excel(path, range = "A2:C11")
test  = read_excel(path, range = "E2:F5")

result = input %>%
  summarize(across(everything(), ~ sum(map_lgl(str_split(.x, ""), ~ all(.x == sort(.x)))))) %>%
  pivot_longer(everything(), names_to = "Words", values_to = "Count")

all.equal(result, test, check.attributes = FALSE)
#> [1] TRUE